cypress-io/cypress-documentation
# Update Dependencies to resolve 404 errors in PRD Build

## 📝 Purpose of Pull Request

> Add a brief description

This updates our dependencies to the latest stable build of Nuxt for our working project. It also introduces a new Webpack Plugin that runs on develop builds to generate a `toc.json` file to test pages for URL Hashes.

## 🤔 How to Setup

### 📄 Page(s) to test

> Please provide links to the changes if applicable

[Home Page](http:localhost:3000/guides/overview/why-cypress)
[Change Log Page](http:localhost:3000/guides/references/changelog)

### ✅ Steps to run pull request

> Please describe the behavior or changes to run this PR

- Update your local by running `yarn`
- Start the server
- Navigate to the Home Page
- Test each section link in the sidenav
- Ensure that the pages render correctly
- Navigate to the Change Log Page 
- Test each header link the right pane
- Ensure that you are navigated to the correct link
